bitcode/devicelib.cl

```diff
@@ -942,6 +942,10 @@ EXPORT float __chip_tex2D_f32(hipTextureObject_t textureObject,
     (float2)(x, y)).x;
 }
 
+OVLD float __int_as_float(int x) { return as_float(x); }
+OVLD int __float_as_int(float x) { return as_int(x); }
+OVLD float __uint_as_float(uint x) { return as_float(x); }
+OVLD uint __float_as_uint(float x) { return as_uint(x); }
```

Comment:
These should follow the naming convention and have  `__chip` prefix and C linkage via `EXPORT`